_rules/2402.md

```diff
@@ -4,18 +4,16 @@ rule_category: layout
 title: Order and group namespaces according to the company
 severity: 3
 ---
-```csharp
-// Microsoft namespaces are first
-using System;
-using System.Collections.Generic;
-using System.Xml;
+	// System namespaces come first.
```

Comment:
🤔 Did you check that coloring remains the same? This is what it looks like now:
<img width="358" alt="image" src="https://user-images.githubusercontent.com/572734/152399249-e0d089f3-60ff-4b3a-9570-0f6bd7e60ca8.png">

Reply:
I removed the csharp marker because all other examples don't use that, and in the produced HTML it renders incorrectly.

![image](https://user-images.githubusercontent.com/10324372/152405124-3a7ac8fb-03ba-4985-b622-bfdfe36e38a5.png)

Reply:
Ok.